src/plugins/shared_ux/public/components/redirect_app_links/redirect_app_links.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent, useRef, useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { ApplicationStart } from 'src/core/public';
+import { createNavigateToUrlClickHandler } from './click_handler';
+
+export interface RedirectAppLinksProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
There's likely a better type for this, so `data-test-subj` and `className` would come along for free.  I'll see what I can find.

Regardless, you could refactor this to be cleaner.  Something like:

```
type Props = React.HTMLAttributes<HTMLDivElement> & Pick<ApplicationStart, 'navigateToUrl' | 'currentAppId$';

export interface RedirectAppLinksProps extends Props {
  className?: string;
  'data-test-subj'?: string;
}
```

Reply:
EUI provides these (and `aria-label`) through `CommonProps` if that's helpful

Reply:
Slightly cleaner now in [dc63e36](https://github.com/elastic/kibana/pull/124197/commits/dc63e36fdc388fbb6c322cf19e71b6be9c91fa35) thanks!